src/index.ejs

```diff
@@ -10,6 +10,7 @@
   <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
   <meta name="theme-color" content="#ffffff">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="//{{.Domain}}/assets/js/piwik.js" async defer></script>
```

Comment:
Hmm, a `script` can't be `async` and `defer` at the same time. We choose to rely on `defer` as it's sufficient for webapps perfs